packages/client/src/lib/hooks/useModal.ts

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react'
```

Comment:
If we need this (see above comment) consider using `unstated`.

Reply:
the reason to use it is to be able to use Modal context's shared value of open or closed here https://github.com/mail-my-ballot/mail-my-ballot/pull/9/files/0b47d6e27264fdaa97be7218862332ec71a35667#diff-1bc1b42e2fd535cf21a00716b7e9d9efR27

So that we can alter the Modal's `isOpen` property across the app. The other way to do this is by using redux.
Does that make sense?